### ISSUE TITLE
bilinovel.com: breakage / detection

### ISSUE THREAD

**Issue by 520HZM** [uBlockOrigin/uAssets#26474]
### Prerequisites

- [x] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [x] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [x] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed or even if you have a different problem.
- [x] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [x] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [x] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [x] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [x] I have turned off all other extensions and the issue still persists. (exception "[Firefox Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers)").
- [x] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site or browser issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.bilinovel.com/novel/2139/76759_8.html`

### Category

breakage

### Description

When using this plugin on mobile Edge, it causes the page to display no text. However, when I disable the element filtering feature, the text reappears, but some blank spaces also appear, which I suspect were originally occupied by ads

### Other extensions used

无

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Chromium Mobile: 131
filterset (summary):
 network: 157528
 cosmetic: 69455
 scriptlet: 23520
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 41131-503, never
  ublock-badware: 12458-1, now
  ublock-privacy: 1655-22, never
  ublock-unbreak: 2545-0, never
  ublock-quick-fixes: 206-0, now
  adguard-mobile: 10014-15, now
  easylist: 78532-520, now
  easyprivacy: 53302-669, now
  urlhaus-1: 23270-0, now
  plowe-0: 3549-83, now
  CHN-0: 25906-105, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 18254 ms
 maxAssetCacheWait: 48 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 0
 extended:
  ##ins.adsbygoogle[data-ad-client]
  ##ins.adsbygoogle[data-ad-slot]
  ##.ads_ad_box
  ##.adsbygoogle-noablate
  ##+js(abort-current-script, document.createElement, pagead2.goog…
```
</details>

**Comment by mapx-** [comment]
```adb
bilinovel.com#@#.ads_ad_box
```

@ryanbr

**Comment by YouGuess11111111** [comment]
Same issue appears again

**Comment by huyszn** [comment]
```adb
bilinovel.com#@#.ads_box
```

@ryanbr

**Comment by zqtg-sudo** [comment]
The issue appears again.

**Comment by mapx-** [comment]
I'm able to access the link using a us ip but redirects to:

`https://www.linovelib.com/novel/2139/76759_8.html`

**Comment by mapx-** [comment]
working link:
`https://www.linovelib.com/novel/4514/261499.html`

detection:

```adb
linovelib.com##+js(noeval-if, /adb/i)
```

**Comment by huyszn** [comment]
I have a US IP and I don't get redirected. The script `https://www.bilinovel.com/scripts/tip_chapter.js?v0220a2` causes the detection on `https://www.bilinovel.com/novel/2139/76759_8.html` and
```adb
bilinovel.com##+js(noeval-if, /adb/i)
```
doesn't work.

**Comment by mapx-** [comment]
I'm still redirected (us vpn) or directly blocked (using my normal european ip)

![Image](https://github.com/user-attachments/assets/ce468d76-9be0-4683-b76d-63992af6b394)

**Comment by huyszn** [comment]
I posted the script in slack

**Comment by mapx-** [comment]
for detection ? if you remove it (rmnt, _0x) does then work ?

**Comment by huyszn** [comment]
Yes for detection. (rmnt, _0x) doesn't work, but blocking the script works, haven't come across breakages yet
```adb
||bilinovel.com/scripts/tip_chapter.js^$script,1p
```

**Comment by mapx-** [comment]
yea, rmnt for external script does not work

**Comment by huyszn** [comment]
> `bilinovel.com##+js(aeld, DOMContentLoaded, _0x)` ?

This works
